mk/aosp_optee.mk

```diff
@@ -86,15 +86,22 @@ $(LOCAL_PREBUILT_MODULE_FILE): $(TA_TMP_FILE)
 	@mkdir -p $(dir $@)
 	cp -uvf $< $@
 
+ifneq ($(local_dependency_ta), )
+$(TA_TMP_FILE): $(TARGET_OUT)/lib/optee_armtz/$(local_dependency_ta)
+endif
```

Comment:
this does not support to set local_dependency_ta with multiple tas, to do that, we could change here as following:
TA_TMP_FILE_DEPS :=
ifneq ($(local_dependency_tas), )
$(foreach dep,$(local_dependency_tas), $(eval TA_TMP_FILE_DEPS += $(TARGET_OUT)/lib/optee_armtz/$(dep)))
endif
$(TA_TMP_FILE: $(TA_TMP_FILE_DEPS)

or something similar.